### DISCUSSION TITLE
Questions about spontaneous SCR

### DISCUSSION THREAD

**Discussion by RyotaTKN** [neuropsychology/NeuroKit#654] [Q&A] [answered]
How can we calculate the frequency of spontaneous SCR?
Is that available by using eda_intervalrelated (i..e, SCR_Peaks_N)?

**Comment by DominiqueMakowski** [comment, marked as answer]
You can detect the location of all peaks (this info should be available in `eda_process()`), then get the interval between them `np.diff(peak_locations)` and from there you can convert to the frequency
